Source/ASControlNode.mm

```diff
@@ -230,6 +238,17 @@ - (void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
                           withEvent:event];
 }
 
+- (void)_cancelTrackingForControlEvents:(ASControlNodeEvent)controlEventMask withEvent:(UIEvent *)event
```

Comment:
I had it first this way, but if you look where we call it, we actually call it also in `-touchesBegan:withEvent:` to have kind of the same event mask in this case as before.